### ISSUE TITLE
Write test to verify installation

### ISSUE THREAD

**Issue by cosmo0920** [cosmo0920/homebrew-mingw_w64#9]
See: https://github.com/cosmo0920/homebrew-mingw_w64/issues/1#issuecomment-173895506

**Comment by tkelman** [comment]
Does the `file` implementation on osx recognize Windows exe's? On Ubuntu 14.04 I get `PE32+ executable (console) x86-64, for MS Windows` from checking a 64 bit exe, or `PE32 executable (console) Intel 80386, for MS Windows` from a 32 bit one.

**Comment by cosmo0920** [comment]
Yes, `file` command on OSX recognizes Windows exe.

Roughly speaking, this test is enough to verify compilers installation place.
e.g. Just confirm existence for `/usr/local/opt/i686-w64-mingw32-gcc/bin/i686-w64-mingw32-gcc` or something else is enough.

**Comment by Luthaf** [comment]
Is that all that is missing for #1 ? I could try to tackle it!

**Comment by cosmo0920** [comment]
> Is that all that is missing for #1 ?

No, homebrew maintainer says that all-in-one formula is preferable to support cross compilers: https://github.com/Homebrew/legacy-homebrew/pull/48385#issuecomment-174442957

**Comment by cosmo0920** [comment]
Following the both of test are acceptable:
- Assert `bin/"i686-mingw-w64-gcc" --version`
- Compile hello world source and execute with Wine.

**Comment by cosmo0920** [comment]
To verify more strict installation checking, we should compile C source with built cross compiler and use wine to verify work built binaries in Windows compatible environment.
